_dashboards/dql.md

```diff
@@ -116,6 +119,36 @@ The [Object fields](#object-fields) and [Nested fields](#nested-fields) sections
 {: .note}
 </details>
 
+## DQL and query string query quick reference
+
+The following table provides a quick reference for both query language commands. 
+
+| Feature | DQL | Query string query (Lucene)|
+|:---|:---|:---|
+| Basic term search | `wind` | `wind` |
+| Multiple terms | `wind gone` (finds documents containing `wind` or `gone`) | `wind gone` (finds documents containing `wind` or `gone`) |
+| Exact phrase search | `"wind rises"` | `"wind rises"` |
+| Field-specific search | `title: wind` | `title:wind` |
+| Existence of a field | `description:*` | `_exists_:description` |
+| Multiple terms in field | `title: (wind OR rises)` <br><br> | `title:(wind OR rises)` |
+| Field containing spaces | `article*title: wind` | `article\ title:wind` |
+| Escaping special characters | `format: 2\*3` | `format:2\*3` |
+| Multiple field search | `title: wind OR description: film` | `title:wind OR description:film` |
+| Nested field search | See [Nested fields](#nested-fields) | Not supported |
+| Numeric range | `page_views >= 100 and page_views <= 300` <br><br> `not page_views: 100` (results include documents with missing `page_views` field) <br><br>   See [Ranges](#ranges)| `page_views:[100 TO 300]` <br><br>  `page_views:(>=100 AND <=300)` <br><br>  `page_views:(+>=100 +<=300)` <br><br>  `page_views:[100 TO *]` <br><br>  `page_views:>=100` <br><br>  `NOT page_views:100` (results include documents with missing `page_views` field) <br><br> See [Ranges]({{site.url}}{{site.baseurl}}/query-dsl/full-text/query-string/#ranges)|
```

Comment:
"results include documents missing the `page_views` field" (both instances)?

Reply:
Reworded